common-lib/src/main/scala/com/gu/mediaservice/model/usage/PrintUsageMetadata.scala

```diff
@@ -41,13 +41,20 @@ case class PrintUsageMetadata(
   type LongElement = List[(String,Long)]
   type IntElement = List[(String,Int)]
 
+  val calculatedPublicationName =
+    if (publicationName.trim.isEmpty) publicationCode match {
+      case "gdn" => "Guardian"
+      case "obs" => "Observer"
+      case _ => "Unknown Publication"
+    } else publicationName
```

Comment:
This is protecting specifically against the case where we try and store only whitespace in a dynamo field which borks, so I don't think it matters if we trim as long as the fields not only whitespace.

Ideally we don't do this at all.